test/capitalization/test_es-pt_capitalization.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Read the input sentence from the file and remove newlines
```

Comment:
Please use the existing regression testing system when adding tests https://wiki.apertium.org/wiki/Apertium-regtest

Reply:
When I use regtest on this corpus, it shows: Failed command: lt-proc -z -w es-pt.automorf.bin. Do you have any idea how to fix this?

Reply:
Since you're running text through the whole pipeline, the input should be plain text, not stream format.